agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -181,7 +182,8 @@ static void fill_son_slave_config(beerocks::config_file::sConfigSlave &beerocks_
     son_slave_conf.stop_on_failure_attempts = 0;
 }
 
-static void son_slave_watchdog(beerocks::config_file::sConfigSlave &beerocks_slave_conf)
+static void son_slave_watchdog(beerocks::config_file::sConfigSlave &beerocks_slave_conf,
+                               const std::unordered_map<int, std::string> &interfaces_map)
```

Comment:
Can be named "interfaces" (nitpick)

Reply:
thank you for the comment. it will not be changed in this PR, but I will consider the suggestion in future commits.